frontend/src/store/NotificationState.ts

```diff
@@ -3,27 +3,78 @@
     It is structured this way so it doesn't stack multiple messages with the same severity (by design).
     It is intended to be used in conjunction with the NotificationStack component.
 */
-import type { Severity } from "@/enum/SeverityEnum";
+
 import { ref } from "vue"
+import { isDesktop, screenSize } from "@/store/screenSizeState";
+import type { Severity } from "@/enum/SeverityEnum";
 
 const defaultNotification = {
```

Comment:
So instead of creating t here new variables (newMsgList, newFormData, newSeverity), we can update the structure of our notification variable.

```
const defaultNotification = {
    success: {msg: "", forestClientNumberList: ""},
    warn: {msg: "", forestClientNumberList: ""},
    error: {msg: "", forestClientNumberList: ""},
}
```

And when you push notification, we store both message and the forest client number list:
```
export const pushNotification = (severity: Severity, message: string, forestClientNumberList: Array<string>) => {
     notifications.value[severity].msg = message;
     notifications.value[severity].forestClientNumberList = forestClientNumberList
 }
```

Reply:
I'll use your suggestion